app/controllers/EpicTestsController.scala

```diff
@@ -34,4 +35,25 @@ class EpicTestsController(authAction: AuthAction[AnyContent], components: Contro
     ),
     fastlyPurger = EpicTestsController.fastlyPurger(stage, ws)
   ) {
+
+  /**
+    * Saves a single test to a new file. The file name will be the test's name.
+    * Overwrites any existing archived test of the same name.
+    * Removing the test from the current active tests list is done separately.
+    */
+  def archive = authAction.async(circe.json[VersionedS3Data[EpicTest]]) { request =>
```

Comment:
@tjsilver I ended up adding this handler to the `EpicTestsController` because it's very specific to epic tests, and so doesn't need to be in a more generic controller

Reply:
good idea 👍